.github/workflows/android.yml

```diff
@@ -31,3 +31,46 @@ jobs:
       run: chmod +x gradlew
     - name: Build with Gradle
       run: ./gradlew build
+      
+ 
+    - name: Run unit tests
+      run: |
+          ./gradlew test --info
+          
+          
+  run-instrumented-tests:
+    runs-on: macos-latest
+    steps:
+      - name: Clone Repo
+        uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3.1.0
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - name: Run instrumented tests
+        uses: reactivecircus/android-emulator-runner@v2.24.0
+        with:
+          api-level: 29
+          script: cd CarEshipApp && ./gradlew connectedAndroidTest
```

Comment:
Try with ./gradlew connectedCheck instead

Reply:
Oh yeah, am gonna that. Thanks.